client/src/components/bonuses.jsx

```diff
@@ -44,16 +59,24 @@ class Bonuses extends React.Component {
 
   render() {
 
-    return (
-      <div className='bonus-tier'>
-        <div className='bonus-title'>
-          <h2>{this.state.bonusTitle}</h2>
+    if (this.state.showWarning) {
+      return (
+        <div className='warning'>
+          Out of range warning! Please choose bundle 1 - 100.
         </div>
-        <div>
-          <BonusItems bonus={this.state.bonus} selectAlbum={this.selectAlbum} selectSong={this.selectSong} />
+      )
+    } else {
+      return (
```

Comment:
Since your `if` block always returns if `true`, you don't need to have this be an `else`, it can just be a `return`.